index.js

```diff
@@ -1,285 +0,0 @@
-    //to make the first layers empty, change the 0 in the second forloop to be j= 3, then it starts at the third layer
-    const row = 13;
-    const col = 25;
-    mapArr = [];
-  
-    for (let i = 1; i < col; i++) {
-       if(!i==0){
-            mapArr.push([]) 
-        }
-      mapArr.push([]);
-      
-      if (i<col-2){
-      for (let j = 0; j < row; j++) { 
-        let tempObj = {ground: randomMaterial(),x:j*50 ,y:i*50};
-        mapArr[i].push(tempObj); }
-   
-     }else if (i == col-2){
-        for (let g = 0; g < row; g++) { 
-            let tempObj = {ground: rock,x:g*50 ,y:i*50};
-            mapArr[i].push(tempObj);}
-     }else{ 
-        for (let k = 0; k < row; k++) { 
-            let tempObj = {ground: gold,x:k*50 ,y:i*50};
-            mapArr[i].push(tempObj);
-            
-        }
-        }
-    }
-  
-    return mapArr;
-  }
-// makeMap/ mapArr to be changed for  generateMap/ matrix at line 56, , 166, 180, 219,220  
-
- 
-function draw(){ 
-    background(220);
-
-   if (showStart){
-    image(startScreen,0,0,0,1000);
-   }
-     else if(gameRunning && goldCount==0 ){
-    textSize(32);
-    text(`Fuel: ${fuel}`,500,35);
-    if(ironCount<5){
-        text(`Iron Collected: ${ironCount}`,50,35);
-    }else {
-        text(`Ironhack activated!`,50,35);
-    }
-    fill(51);
-    
- for(let i=0;i<mapArr.length;i++){
-        let tempArr = mapArr[i];
-        for (let j=0; j<tempArr.length;j++){
-             image(tempArr[j].ground,tempArr[j].x,tempArr[j].y);
-        }
-    }
-
- image (playerNorm,player.x,player.y);
-
-   if (fuel < 0 && !goldCount>=1){
-        resetButton = false;
-        gameRunning = false;
-    }
-    //out of fuel
-    }else if(showStart==false &&gameRunning== false && goldCount==0){
-    image(gameOverScreen,0,0,0,1000);
-    const button = document.querySelector('#restart');
-     button.style.cssText= `visibility: visible`;
-    }
-    // win screen
-    else if(goldCount>=1){
-        resetButton= false;
-        gameRunning= false;
-        image(winScreen,0,0,0,1000);
-        const button = document.querySelector('#restart');
-        button.style.cssText= `visibility: visible`;
-    }
-}
-function startGame (){
-    resetButton=true;
-    gameRunning=true;
-    showStart = false;
-    
-    document.querySelector('#start').style.cssText = `visibility: hidden`;
-   //generateMap();
-    makeMap();
-
-}
-function resetGame(){
-    resetButton=true;
-    gameRunning=true;
-    player.x = 300;
-    player.y = 0;
-    fuel = 15;
-    goldCount = 0;
-    ironCount =0;
-    
-    document.querySelector('#restart').style.cssText = `visibility: hidden`;
-    //generateMap();
-    makeMap();
-
-}
-
-function randomMaterial(){
-    let rand= randomNum(100);
-    if (rand>=9){
-        return dirt;
-    }else if(rand<=4){
-        return iron;
-    }else{
-        return oil;
-    }
-}
-
-function randomNum(max){
```

Comment:
in p5 there is a random function that you can give min and max already built in :) 
so no need to rewrite it :+1: